src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/CourtCaseEntity.java

```diff
@@ -158,12 +85,6 @@
     @Column(name = "manual_update", nullable = false, updatable = false)
     private boolean manualUpdate;
 
-    @Deprecated(forRemoval = true)
-    @PrePersist
-    public void isManualUpdate(){
```

Comment:
Was this used anywhere do you know? I presume not or it would have broken stuff, happy for it to remain deleted as long as we're confident.